gradle-baseline-java/src/main/groovy/com/palantir/baseline/tasks/CheckJUnitDependencies.java

```diff
@@ -143,7 +143,7 @@ private boolean sourceSetMentionsJUnit4(SourceSet ss) {
     }
 
     private boolean sourceSetMentionsJUnit5Api(SourceSet ss) {
-        return !ss.getAllSource()
+        return !ss.getAllJava()
```

Comment:
does this scan Groovy, Scala, Kotlin, etc. or only `**/*.java`?

Reply:
It does support
	groovy: https://github.com/gradle/gradle/blob/master/subprojects/plugins/src/main/java/org/gradle/api/plugins/GroovyBasePlugin.java#L110
	kotlin: https://github.com/JetBrains/kotlin/blob/master/libraries/tools/kotlin-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/gradle/plugin/KotlinPlugin.kt#L116
	scala: https://github.com/gradle/gradle/blob/master/subprojects/scala/src/main/java/org/gradle/api/plugins/scala/ScalaBasePlugin.java#L122